willowlab/tests/__init__.py

```
@@ -0,0 +1,9 @@
+"""Test helpers for the willowLab suite."""
+import importlib
+import sys
+
+try:
+    import numpy  # type: ignore # noqa: F401
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)